simpleclient_spring_boot/src/main/java/io/prometheus/client/spring/boot/ExportMetricsToPrometheus.java

```diff
@@ -0,0 +1,17 @@
+package io.prometheus.client.spring.boot;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import org.springframework.context.annotation.Import;
+
+
+@Target(ElementType.TYPE)
+@Retention(RetentionPolicy.RUNTIME)
+@Documented
+@Import(PrometheusMetricsConfiguration.class)
+public @interface ExportMetricsToPrometheus {
```

Comment:
This name doesn't really capture what it does. `EnableSpringBootMetricsCollector` would be better I think.